client/src/components/pages/jobs/FakeDataForJobs.tsx

```diff
@@ -0,0 +1,113 @@
+
+const fakeData = [
+  {
+    id: "1",
+    title: "Software Engineer",
+    company_name: "Example Corp",
+    location_city: "San Francisco",
+    work_location: "Remote",
+    employment_type: "Full-time",
+    skills: [
+      { skill_id: "1", skill_name: "JavaScript" },
+      { skill_id: "2", skill_name: "React" },
+      { skill_id: "3", skill_name: "Node.js" },
+    ],
+    soft_skills: ["Teamwork", "Problem Solving"],
+    matchScore: 75,
+    isSaved: false,
+    associations: ["Association 1", "Association 2"],
```

Comment:
why this file is committed?

Reply:
I needed it for testing it shouldn’t be there but I did git add . therefore everything is pushed

Reply:
then please delete it